package/gluon-wan-dnsmasq/files/etc/hotplug.d/iface/50-gluon-wan-dnsmasq

```diff
@@ -1,3 +1 @@
-if [ "$INTERFACE" = 'wan' -o "$INTERFACE" = 'wan6' ]; then
-   /lib/gluon/wan-dnsmasq/update.lua
-fi
+/lib/gluon/wan-dnsmasq/update.lua "$INTERFACE"
```

Comment:
Removing the `$INTERFACE` checks will cause way more Lua processes to be spawned when setting up the network (or when tearing it down), so I'd like to keep it.

Reply:
My intention was to allow additional packages to only drop a file to `/lib/gluon/wan-dnsmasq/interface.d/`. But i see the performance impact this implies.

Will restore the previous behavior.